.travis.yml

```diff
@@ -121,10 +134,49 @@ jobs:
         - npm run integration-test-run-basic || { npm run integration-test-cleanup; exit 1; }
         - npm run integration-test-run-all || { npm run integration-test-cleanup; exit 1; }
         - npm run integration-test-cleanup
+        - |
+          tagName=`git diff HEAD^ package.json | grep '"version": "' | tail -n 1 | grep -oE "[0-9]+\.[0-9]+\.[0-9]+"` || :
+          if [ -n "$tagName" ]
+          then
+            git tag v$tagName
+            git push https://$GITHUB_TOKEN@github.com/serverless/serverless --tags
+            if [ $TRAVIS_BRANCH = master ]
+            then
+              # Ensure `release-fast-track` branch points latest release
+              git push -f https://$GITHUB_TOKEN@github.com/serverless/serverless master:release-fast-track
+            elif [ $TRAVIS_BRANCH = release-fast-track ]
+            then
+              # Fast forward to `master`
+              git fetch --unshallow
+              git config remote.origin.fetch "+refs/heads/*:refs/remotes/origin/*"
+              git fetch origin master
+              git rebase origin/master
+              git checkout master
+              git merge release-fast-track
```

Comment:
@pmuens and @medikoo This comment is not related to this PR.. My changes which were supposed to go in 1.59.1 were not included.

Since this looks like next release I wanted to bring above to your notice. Please make sure #7040 changes are packaged into next release..

If you're wondering what happend here are the logs.

https://travis-ci.org/serverless/serverless/builds/621107104?utm_source=github_status&utm_medium=notification

https://travis-ci.org/serverless/serverless/jobs/621107105?utm_medium=notification&utm_source=github_status

Thanks for your help.

Reply:
@avinasherupaka we're aware of your PR (no need to put unrelated comments elsewhere).

We always let features settle a bit before releasing, It'll be released with v1.60.0

Reply:
> If you're wondering what happend here are the logs.

This logs are result of internal testing on our side of new release flow, you can ignore them